### PR TITLE
Add missing channel getters

### PR DESCRIPTION
**Pull request by JakubKohout**
Some properties in `ChannelData` weren't accessible trough `ChannelAPI`

### CODE REVIEW

File: src/API/Channel/Channel.ts [hunk 1]
```diff
@@ -48,4 +48,73 @@ export default class Channel extends ChannelPlaceholder {
...
+	get broadcasterLanguage() {
+		return this._data.broadcaster_language;
+	}
+
+	get broadcasterType() {
+		return this._data.broadcaster_type;
+	}
+
+	get createdAt() {
+		return this._data.created_at;
```
[the remark this answers is not in the document]
Comment (JakubKohout) [reply src/API/Channel/Channel.ts]: 👍 good idea

File: src/API/Channel/Channel.ts [hunk 2]
```diff
@@ -48,4 +48,73 @@ export default class Channel extends ChannelPlaceholder {
...
+
+	get profileBannerBackgroundColor() {
+		return this._data.profile_banner_background_color;
+	}
+
+	get status() {
+		return this._data.status;
+	}
+
+	get updateAt() {
```
Comment (d-fischer) [inline src/API/Channel/Channel.ts]: Small nitpick: updatedAt, not updateAt

Comment (JakubKohout) [reply src/API/Channel/Channel.ts]: 👍

**Comment by d-fischer** [comment]
Thanks for your effort! Merged :)

**Comment by d-fischer** [comment]
@all-contributors please add JakubKohout for code

**Pull request merged**